gartan_fetch.py

```
@@ -226,6 +226,26 @@ class AuthenticationError(Exception):
+    Fetches the HTML content of the station feed display.
+
+    Args:
+        session (requests.Session): An authenticated session.
+
+    Returns:
+        str: The HTML content of the page, or None if the request fails.
+    """
+    try:
+        response = session.get(STATION_FEED_URL)
+        response.raise_for_status()  # Raise an exception for bad status codes
+        return response.text
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)